libs/integration-interface/README.md

```diff
@@ -0,0 +1,7 @@
+# integration-interface
+
+This library contains all the different Topic instances.
```

Comment:
It contains everything that is needed to communicate with the shell and that is framework-independent